packages/rdfine-data-cube/Table/Table.ts

```diff
@@ -88,7 +89,7 @@ export function FactTableMixin<TBase extends Constructor> (Base: TBase) {
       }
 
       const path = [
-        this.name,
+        urlSlug(this.name),
```

Comment:
Should this fix also be applied to the `aboutUrl` property of the mapping?

Reply:
Could you be more specific?

Reply:
Forget it, I think I'm confusing stuff.